bootstrap/v2/pkg/kfapp/kustomize/kustomize.go

```diff
@@ -1002,6 +1002,8 @@ func GenerateKustomizationFile(kfDef *kfdefsv2.KfDef, root string,
 		kustomization.CommonLabels = map[string]string {
 			kftypesv2.DefaultAppLabel: kfDef.Name,
 		}
+	} else {
```

Comment:
this may not work since we are installing multiple applications
See #3429 

Were you able to test in the case where only k8s are installed?

Reply:
I had tested it by "kfctl generate/apply k8s", but `kftypesv2.DefaultAppLabel` label is absent for all resources.
Thus we cannot delete global resources when I run "kfctl delete"

Reply:
Refer to https://github.com/kubeflow/kubeflow/blob/master/bootstrap/v2/pkg/kfapp/kustomize/kustomize.go#L382-L406

Reply:
The selector `kftypesv2.DefaultAppLabel: kfDef.Name` can be used to match resources that belong to the Application when applicationEnabled.
Anyway, I think we should make sure this common label added.